index.html

```diff
@@ -135,11 +135,14 @@ <h4 class="fragment green" data-fragment-index="1">Ansible/Puppet/Chef or config
 
             <section>
                 <p>
-                    Since openssh 6.2 (2013-03-22) a new option is available: <em>AuthorizedKeysCommand</em>
+                  Since openssh 6.2 (2013-03-22) a new option is available
                 </p>
                 <p class="fragment" data-fragment-index="1">
-                    it supports <em>fetching authorized_keys from a command in addition to (or
-                    instead of) from the filesystem.</em>
```

Comment:
This sentence is a citation from the release note, I'd keep it as it is

Reply:
Indeed I thought it was and looked into the manpage but didn't find it there. Still "in addition to from the filesystem" sounds really bad to me. I'd personally drop the quote, but that's up to you! ;-)